src/H5Fint.c

```diff
@@ -38,6 +38,10 @@
 #include "H5Tprivate.h"  /* Datatypes                                */
 #include "H5VLprivate.h" /* Virtual Object Layer                     */
 
+#if 1 /* JRM */ /* probably want to re-work this */
```

Comment:
This will not be required when the SWMR VFD is refactored into a pass-through driver.